src/SFA.DAS.AssessorService.Data/EMailTemplateQueryRepository.cs

```diff
@@ -1,24 +1,30 @@
-﻿using System.Collections.Generic;
-using System.Linq;
+﻿using Dapper;
 using System.Threading.Tasks;
-using Microsoft.EntityFrameworkCore;
 using SFA.DAS.AssessorService.Application.Interfaces;
-using SFA.DAS.AssessorService.Domain.Entities;
+using SFA.DAS.AssessorService.Domain.DTOs;
+using System.Linq;
 
 namespace SFA.DAS.AssessorService.Data
 {
-    public class EMailTemplateQueryRepository : IEMailTemplateQueryRepository
+    public class EMailTemplateQueryRepository : Repository, IEMailTemplateQueryRepository
     {
-        private readonly AssessorDbContext _assessorDbContext;
-
-        public EMailTemplateQueryRepository(AssessorDbContext assessorDbContext)
+        public EMailTemplateQueryRepository(IUnitOfWork unitOfWork) : base(unitOfWork)
         {
-            _assessorDbContext = assessorDbContext;
         }
 
-        public async Task<EMailTemplate> GetEmailTemplate(string templateName)
+        public async Task<EmailTemplateSummary> GetEmailTemplate(string templateName)
         {
-            return await _assessorDbContext.EMailTemplates.FirstOrDefaultAsync(q => q.TemplateName == templateName);
+            var sql = @"SELECT ET.Id, ET.TemplateName, ET.TemplateId, ETR.Recipients, ET.RecipientTemplate
+                    FROM [dbo].[EMailTemplates] ET join [dbo].[EmailTemplatesRecipients] ETR
```

Comment:
should use inner join here explicitly (join is the same thing but inner is clearer)